### DISCUSSION TITLE
Rich Text in broadcast

### DISCUSSION THREAD

**Discussion by TeemoCell** [SmartlyDressedGames/Unturned-3.x-Community#4378] [Ideas]
Would be nice if you ad rich text support to the `broadcast` command.

Currently not supported or i use wrong tags :D
![TeCFY5jo5](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/11705513/d4a4ff7d-9e99-4bba-980a-5bf1cb234eae)

**Comment by GazziFX** [comment]
Why not create a new command if you're using plugins

**Reply by TeemoCell** [reply]
I want to use less plugins. The base command is already there. It just don't support rich text.

**Reply by GazziFX** [reply]
Not sure how it will exactly work, always use it as rich may break compatibility, additional arguments will be inconvenient to use

**Reply by DanielWillett** [reply]
Also `/broadcast` is in rocketmod/ldm so may be better to put this in that repo. Or RocketModFix since nelson isn't adding new features to ldm as far as I know.

**Reply by Jdance-Media** [reply]
Well, it would also be nice to have rich text support for the `/say` vanilla command.

**Reply by TeemoCell** [reply]
> Also `/broadcast` is in rocketmod/ldm so may be better to put this in that repo. Or RocketModFix since nelson isn't adding new features to ldm as far as I know.

Oh okay. Didn't know it was from ldm.

**Reply by TeemoCell** [reply]
> Well, it would also be nice to have rich text support for the `/say` vanilla command.

Yeah I thought broadcast was vanilla. That's why I posted it here.
